docs/source/configuration/entity-caching.mdx

```diff
@@ -0,0 +1,126 @@
+
+For example, consider the `Products` and `Inventory` subgraphs from the [Entities guide](/federation/entities):
+
+<CodeColumns>
+
+```graphql title="Products subgraph"
+type Product @key(fields: "id") {
+  id: ID!
+  name: String!
+  price: Int
+}
+```
+
+```graphql title="Inventory subgraph"
+type Product @key(fields: "id") {
+  id: ID!
+  inStock: Boolean!
+}
+```
+
+</CodeColumns>
+
+Assume we have a client for a shopping cart application, and it makes a query that requests the following for each product in the cart:
+- The product's name and price. This data is from the `Products` subgraph.
+- The product's availability in inventory. This data is from the `Inventory` subgraph.
+
+If we were caching the entire client response, we would have to store it with a short time to live, because the cart data can change often and the real-time inventory has to be up to date. A client-response cache couldn't be shared between users, because the cart is personal. We might also duplicate a lot of data between client responses, because a lot of the products might appear in multiple carts.
+
+With entity caching enabled for this example, the router is able to:
+- Store each product's description and price separately with large TTL.
+- Minimize the number of subgraph queries made by each client query, with potentially some client queries making zero subgraph queries and instead fetching all product data from cache.
+- Share the product cache between all users.
+- Cache the cart per user, with a small amount of data.
+- Cache inventory data with a low TTL or not cache it at all.
+
+## Use entity caching
+
+Follow this guide to enable and configure entity caching in the Apollo Router.
+
+### Prerequisites
+
+To use entity caching in the Apollo Router, you must set up:
+
+- A Redis instance or cluster that your router instances can communicate with.
+- A [GraphOS Enterprise plan](https://www.apollographql.com/pricing/) that [connects your router to GraphOS](./overview/#environment-variables).
+
+### Configure router for entity caching
+
+In `router.yaml`, configure `experimental_entity_cache`:
+- Enable entity caching globally.
+- Configure Redis, using the same conventions described in [distributed caching](./distributed-caching).
+- Configure entity caching per subgraph, with overrides per subgraph for disabling entity caching and TTL.
+
+For example:
+
+```yaml title="router.yaml"
+experimental_entity_cache:
+  enabled: true
+
+  redis:
+    urls: ["redis://..."]
+    timeout: 5ms # Optional, by default: 2ms
+    ttl: 24h # Optional, by default no expiration
+
+  subgraphs:
+    accounts:
+      enabled: false # disable for a specific subgraph
+    reviews:
+      ttl: 120s # overrides the global TTL
+```
```

Comment:
It may be helpful to add/align a comment for each to the bullet points above to the config and continue with the example given above:

```suggestion
```yaml title="router.yaml"
# Enable entity caching globally
experimental_entity_cache:
  enabled: true

  # Configure Redis
  redis:
    urls: ["redis://..."]
    timeout: 5ms # Optional, by default: 2ms
    ttl: 24h # Optional, by default no expiration

  # Configure entity caching per subgraph
  subgraphs:
    products:
      ttl: 120s # overrides the global TTL
    inventory:
      enabled: false # disable for a specific subgraph
```
```